scripts/leapfrog/playbooks/elasticsearch-postleap-pip-upgrade.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This playbook needs the boiler plate license for RPC-OpenStack

Reply:
yes, you're right, adding it.

Reply:
yes, you're right, adding it.